### PR TITLE
feat(aggregations): add keyboard shortcuts COMPASS-6391

### PR DESCRIPTION
**Pull request by mabaasit**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx [hunk 1]
```diff
@@ -82,29 +153,39 @@ export const FocusModeModalHeader: React.FunctionComponent<
...
-            size="xsmall"
-            title={null}
-            role="presentation"
-            glyph="ChevronLeft"
-          ></Icon>
-        </Button>
-
-        {/* @ts-expect-error leafygreen unresonably expects a labelledby here */}
+          <Body className={tooltipContentStyles}>
+            <span>Goto Previous Stage</span>
```
Comment (gribnoysup) [inline packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx:178]: This is label / description, not a title

```suggestion
            <span>Go to previous stage</span>
```

File: packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx [hunk 2]
```diff
@@ -118,21 +199,32 @@ export const FocusModeModalHeader: React.FunctionComponent<
...
         >
-          <Icon
-            size="xsmall"
-            title={null}
-            role="presentation"
-            glyph="ChevronRight"
-          ></Icon>
-        </Button>
+          <Body className={tooltipContentStyles}>
+            <span>Goto Next Stage</span>
```
Comment (gribnoysup) [inline packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx:224]: Ditto here

```suggestion
            <span>Go to next stage</span>
```

File: packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx [hunk 3]
```diff
@@ -82,29 +153,39 @@ export const FocusModeModalHeader: React.FunctionComponent<
...
+          </Body>
+        </Tooltip>
+
         <Select
           allowDeselect={false}
           style={stageSelectStyle}
           size="xsmall"
           value={String(stageIndex)}
-          aria-label="Select stage to edit"
+          aria-labelledby="Select stage to edit"
```
[the remark this answers is not in the document]
Comment (mabaasit) [reply packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx]: added it back. very irritating that it throws.

File: packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx [hunk 4]
```diff
@@ -47,12 +49,28 @@ const fakeToggleLabelStyles = css({
...
 
+const menuStyles = css({
+  width: '240px',
+});
+
 const menuItemStyles = css({
   '&:after': {
     content: 'attr(data-hotkey)',
+    position: 'absolute',
+    right: 20, // LG has 20px padding on the right
```
Comment (gribnoysup) [inline packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx:60]: Or you can set white-space to nowrap and then you don't need remember to change these styles when leafygreen inevitably changes paddings that we don't control 🙂

**Review by gribnoysup — APPROVED** [review]

**Pull request merged**